notebooks/VAD

```diff
@@ -0,0 +1,37 @@
+from scipy import signal
+# input : y - the signal, sr - sample rate
+# output : start_times - vector of start times of the sound events
+#          end_time - vector of end times of the sound events            
+def detect_activity(y, sr,
+        n_mels=128, fmin=1000, fmax=11025, hop_length=512,
+        gain=0.8, bias=10, power=0.25, pcen_time_constant=0.06, eps=1e-06,
+        medfilt_time_constant=1, normalized =true,
```

Comment:
`medfilt_time_constant` should default to `None` and be expressed in seconds, not time frames.

Reply:
`normalized =true` -> `normalized=True`